config/checker/git4idea.astub

```diff
@@ -0,0 +1,34 @@
+import org.checkerframework.checker.tainting.qual.Untainted;
+import org.jetbrains.annotations.Nls;
+import com.intellij.openapi.util.NlsContexts.DialogTitle;
```

Comment:
Ditto, remove unused annotations, TBD if `NotNull` is redundant or not; `Nullable` definitely is NOT redundant

Reply:
Plus add `@Tainted` for clarity where applicable